src/uct/ib/mlx5/dc/dc_mlx5.inl

```diff
@@ -65,13 +65,12 @@ uct_dc_mlx5_ep_pending_common(uct_dc_mlx5_iface_t *iface, uct_dc_mlx5_ep_t *ep,
     UCS_STATIC_ASSERT(sizeof(uct_dc_mlx5_pending_req_priv) <=
                       UCT_PENDING_REQ_PRIV_LEN);
 
-    if (uct_dc_mlx5_iface_is_policy_shared(iface)) {
+    if (ep->dci != UCT_DC_MLX5_EP_NO_DCI) {
```

Comment:
Why changed? seems the access to uct_dc_mlx5_pending_req_priv(req)->ep is still done only when uct_dc_mlx5_iface_is_policy_shared() is true, in uct_dc_mlx5_ep_arbiter_purge_cb.
Is there another usage?

Reply:
Yes, because we want to use "shared policy" logic also when in hybrid mode

Reply:
why? where is this field used in hybrid mode?